### PR TITLE
Css refactoring

### PR DESCRIPTION
**Pull request by ReenaRajani**
Updates based on the comments as discussed

### CODE REVIEW

File: css/materialize.css [hunk 1]
```diff
@@ -8583,6 +8583,7 @@ a.waves-effect .waves-ripple {
   padding: 0 12px;
   border-radius: 16px;
   background-color: #e4e4e4;
+  margin: 10px 0px;
```
Comment (ButenkoT) [inline css/materialize.css]: When we have a someone's library we are not changing things in it. That is for we have our own css where we can overwrite some particular pieces  that are suitable for us. Can you move it to `styles.css` file, please. 
`.chip{
 margin: 10px 0px;
}`

File: contacts.html [hunk 2]
```diff
@@ -90,7 +90,7 @@ <h3 class="header center green-text">Contact us</h3>
                         <div class="response" id="mce-success-response" style="display:none"></div>
                     </div>    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
                     <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_83a95d4fc93cca6797b6b6832_208855fbfd" tabindex="-1" value=""></div>
-                    <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
+                    <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button" style="margin:0.5em 1em"></div>
```
Comment (ButenkoT) [inline contacts.html]: would be better if you do it in `style.css` file, so we do not touch 3d party code.

**Review by ButenkoT — CHANGES_REQUESTED** [review]

**Comment by ReenaRajani** [comment]
@ButenkoT . this is done

**Review by ButenkoT — APPROVED** [review]

**Comment by ButenkoT** [comment]
All looks good, I guess we can merge it

**Pull request merged**